src/frontend/apps/impress/src/features/header/components/Header.tsx

```diff
@@ -1,89 +1,85 @@
 
-import { Box, StyledLink, Text } from '@/components/';
+import { Box, Icon, StyledLink, Text } from '@/components/';
 import { ButtonLogin } from '@/core/auth';
+import { useCunninghamTheme } from '@/cunningham';
 import { LanguagePicker } from '@/features/language';
 import { useResponsiveStore } from '@/stores';
 
 import { default as IconDocs } from '../assets/icon-docs.svg?url';
+import { HEADER_HEIGHT } from '../conf';
 
-import { DropdownMenu } from './DropdownMenu';
 import { LaGaufre } from './LaGaufre';
 
 export const Header = () => {
   const { t } = useTranslation();
-  const { isSmallMobile } = useResponsiveStore();
+  const theme = useCunninghamTheme();
+
+  const spacings = theme.spacingsTokens();
+  const colors = theme.colorsTokens();
+  const { isDesktop, toggleMobileMenu } = useResponsiveStore();
 
   return (
     <Box
       as="header"
-      $justify="center"
-      $width="100%"
-      $zIndex="100"
-      $padding={{ vertical: 'xtiny' }}
-      $css="border-bottom: 1px solid #EDEDED;"
+      $css={css`
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        height: ${HEADER_HEIGHT}px;
+        min-height: ${HEADER_HEIGHT}px;
+        padding: 0 ${spacings['base']};
+        background-color: ${colors['greyscale-000']};
+        border-bottom: 1px solid ${colors['greyscale-200']};
+      `}
     >
-      <Box
-        $margin={{
-          left: 'big',
-          right: isSmallMobile ? 'none' : 'big',
-        }}
-        $align="center"
-        $justify="space-between"
-        $direction="row"
-      >
-        <Box>
-          <StyledLink href="/">
-            <Box
-              $align="center"
-              $gap="0.8rem"
-              $direction="row"
-              $position="relative"
-              $height="fit-content"
-              $margin={{ top: 'auto' }}
-            >
-              <Image priority src={IconDocs} alt={t('Docs Logo')} width={25} />
-              <Text
-                $padding="2px 3px"
-                $size="8px"
-                $background="#368bd6"
-                $color="white"
-                $position="absolute"
-                $radius="5px"
-                $css={`
-                  bottom: 13px;
-                  right: -17px;
-                `}
-              >
-                BETA
-              </Text>
-              <Text
-                $margin="none"
-                as="h2"
-                $color="#000091"
-                $zIndex={1}
-                $size="1.30rem"
-                $css="font-family: 'Marianne'"
-              >
-                {t('Docs')}
-              </Text>
-            </Box>
-          </StyledLink>
+      {!isDesktop && (
+        <Button
+          size="medium"
+          onClick={toggleMobileMenu}
+          aria-label={t('Open the header menu')}
+          color="primary-text"
+          icon={<Icon iconName="menu" />}
```

Comment:
If you don't use the Burger component anymore, you should remove it from the repo (another commit maybe).